tests/cpu/test_cca.cpp

```diff
@@ -23,6 +23,8 @@
 // System include(s).
 #include <functional>
 
+/*
+@FIXME: Change the csv data format for csv
```

Comment:
This is pretty bad. :frowning: How come we can't read both cell data types now? Then why does the current `main` branch read the new CSV cell files without errors? (Even if one of the columns changed names.)

Reply:
I don't know. Feel free to fix the PR if you find the reason.
This is what I get when running the test
```
[  FAILED  ] SparseCclAlgorithm/ConnectedComponentAnalysisTests.Run/trackml_like_0000000028, where GetParam() = (32-byte object <10-FE 29-01 00-00 00-00 70-DB 2B-01 00-00 00-00 E0-51 46-00 00-00 00-00 20-59 46-00 00-00 00-00>, "trackml_like_0000000028") (0 ms)
[ RUN      ] SparseCclAlgorithm/ConnectedComponentAnalysisTests.Run/trackml_like_0000000029
unknown file: Failure
C++ exception with description "Missing header column 'measurement_id'" thrown in the test body.
```

Reply:
I also don't see the error in the `main` branch, but the `hit_id` is always zero which drops the efficiency to zero as you saw in the mattermost.

Reply:
Let's keep this one open for the moment. :thinking: I'll try it out shortly from your branch.

Worst case scenario, we make some measurements using this branch as a temporary solution, and then we discuss how to implement all of these things in a slightly more pleasing way. :thinking:

Reply:
Sure